man/tuned-adm.8

```diff
@@ -110,6 +110,16 @@ Enable automatic profile selection mode, switch to the recommended profile.
 .B profile_mode
 Show current profile selection mode.
 
+.TP
+.B "instance_acquire_devices \fI[\-c | \-\-cpu\-list]\fP \fIdevices\fP \fIinstance\fP"
+Move all provided \fIdevices\fP under the given plugin \fIinstance\fP. The plugin
+instance is identified using its name from the TuneD profile.
+
+The devices are specified using a comma-separated list. When the \fI\-c\fP
+or the \fI\-\-cpu\-list\fP flag is enabled, the tool assumes that CPU devices
+are specified using a list of their numbers, such as "0,2-4", meaning that
```

Comment:
It's not just a list, it has also ranges, e.g. 2-4. The syntax [1] is called cpulist, sometimes also cpu list, CPU list or cpu-list, CPU-list, the convention is not clear. In TuneD for consistence we usually use the term "cpulist".

Recently the group modifiers were added to the syntax (:size/group) and N which we do not support yet in TuneD, but I probably wouldn't mention this maybe temporal limitation in the man.

[1] https://www.kernel.org/doc/html/next/admin-guide/kernel-parameters.html#cpu-lists